utils/install.py

```diff
@@ -1317,6 +1338,7 @@ def __init__(
         self.ld_library_path = None
         self.dist = dist_
         self.release_package_wasmedge = None
+        self.cuda = check_nvcc(self.platform)
```

Comment:
You might want to move this later in the code

Reply:
Sure. Let's do this in another PR.